analysis/wwz/get_wwz_yields.py

```diff
@@ -590,30 +591,30 @@ def make_syst_plots(histo_dict,grouping_mc,grouping_data,save_dir_path,year):
                 mc_down_arr = mc_down[{"process_grp":sum}].values()
 
                 # Print individual syst numbers
-                #if var_name != "nleps": continue
-                #n = sum(sum(mc_nom.values()))
-                #u = sum(mc_up_arr)
-                #d = sum(mc_down_arr)
-                #print("\n",syst)
-                #print("nom",n)
-                #print("up",u)
-                #print("do",d)
-                #r_up = abs((n-u)/n)
-                #r_do = abs((n-d)/n)
-                #r = (r_up+r_do)/2
-                #print("err",np.round(100*abs(n-u)/n,1),"%")
-                #print("err up",np.round(100*r_up,1),"%")
-                #print("err do",np.round(100*r_do,1),"%")
+                if var_name != "nleps": continue
```

Comment:
Can we leave this block (from here through the continue statement) commented by default? Otherwise I think this function won't make any plots by default. 

Eventually might be nice to have an option to turn on and off this print stuff. But probably don't need it right now.